src/compiler/types.ts

```diff
@@ -4968,7 +4968,6 @@ export interface TypeChecker {
     getWidenedLiteralType(type: Type): Type;
     /** @internal */
     getPromisedTypeOfPromise(promise: Type, errorNode?: Node): Type | undefined;
-    /** @internal */
     getAwaitedType(type: Type): Type | undefined;

```

Comment:
Can you copy the docstring from `getAwaitedType` in the checker here?

Reply:
@jakebailey  sure! https://github.com/microsoft/TypeScript/pull/59268/commits/0232440d8d0883bf828c7b8fe7f8f230405bbc5b